.github/workflows/latest_image_push.yml

```diff
@@ -12,12 +12,13 @@ jobs:
       - name: Checkout Code
         uses: actions/checkout@v2
 
-      - name: Set up Docker Buildx
-        id: buildx
-        uses: crazy-max/ghaction-docker-buildx@v3.3.1
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1.0.1
```

Comment:
What is QEMU version? Is it configurable separately?

Reply:
Unfortunately, the only configurable fields are `image` and `platforms`. Default image is : `tonistiigi/binfmt:latest`.

Reply:
I see. It seems latest image utilizes QEMU 5.0.0 currently:

<img width="834" alt="Screen Shot 2021-01-07 at 14 35 14" src="https://user-images.githubusercontent.com/6005622/103888323-9b70c280-50f5-11eb-93fc-3afc1813141b.png">